plugins/modules/katello_sync_plan.py

```diff
@@ -80,85 +80,70 @@
     organization: "Default Organization"
     interval: "weekly"
     enabled: false
-    sync_date: "2017/01/01 00:00:00 +0000"
+    sync_date: "2017-01-01 00:00:00 UTC"
```

Comment:
should the expected date format be documented somewhere?

Reply:
This example is part of the documentation.
I think, katello understands other formats, too. It's just not idempotent then.